spec/unit/rbeapi/api/alias/fixture_alias.text

```diff
@@ -0,0 +1,3 @@
+alias Alias1 my command
```

Comment:
Martin,
  Please ensure the regexes can handle multiline aliases as in the following:
```
!
prompt %H%R%v%P
alias sib show ip interface brief
no service configuration session max completed
no service configuration session max pending
!
alias desc
   !! Syntax: desc <interface> <description>
   1 conf
   2 int %1
   3 description %2
   4 end
!
```

One possible solution, while not terribly pretty, would be the following:
```
node.running_config.scan(/^alias (\w+)((?:(?= )(?:.+?)(?=\n)|\n(?:.+?)(?=\n\!)))/m)
=> [["sib", " show ip interface brief"],
 ["desc",
  "\n   !! Syntax: desc <interface> <description>\n   1 conf\n   2 int %1\n   3 description %2\n   4 end"]]
```

The initial \n in the multiline body is both a good indicator that it is a multiline alias.  You may still need to do a split before passing it to `node.config()` though.